### DISCUSSION TITLE
请问这个函数radius的变量circle的属性是什么？

### DISCUSSION THREAD

**Discussion by 0xsu** [NonceGeek/Move-Camp#241] [Q&A]
<img width="738" alt="image" src="https://user-images.githubusercontent.com/101964290/212307622-1be37cfb-e823-4f85-b2bf-b6738c00502e.png">

**Comment by Demian101** [comment]
radius 是一个函数，其作用是获取某个特定圆的半径
Circle 是一个”圆结构体“，定义了世界上的各种圆
circle 是某个特定圆，比如半径为 3m 的圆
pub fun radius(circle: &Circle) 表示给 radius 函数传入某个特定圆的实例的引用（reference），获取其半径并返回
